blazor/dropdown-list/popup-setting.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title: Popup Setting in Blazor DropDown List Component | Syncfusion
+description: Checkout and learn here all about Popup Setting in Syncfusion Blazor DropDown List component and much more.
+platform: Blazor
+control: DropDown List
+documentation: ug
+---
+
+# Popup Setting in Dropdown List
+
+## Change the popup width
+
+You can customize the width of the popup using [PopupWidth](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_PopupWidth). Default value of `PopupWidth` is `100%`.

```

Comment:
use this content - You can customize the width of the popup using [PopupWidth](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_PopupWidth). Default value of `PopupWidth` is `100%` so the popup width sets based on the width of the DropdownList component.